src/modules/announcements/announcement.controller.ts

```diff
@@ -61,9 +75,12 @@ export class AnnouncementController {
     );
   }
 
-  // Delete a Announcement
+  // Delete an Announcement by Id
   @Delete('/:announcementId')
   @ApiParam(announcementId)
+  @ApiOperation({
+    summary: 'Delete an Announcement by Id',
```

Comment:
Don't end sentence with `,` comma otherwise it will become multiline by `eslint fix`.

Reply:
Fixed 🚀